### PR TITLE
[POC] Dequipper

### PR DESCRIPTION
**Pull request by adamhannigan**
# Description

Dequip wearables on additional Bumpkins and send to your personal wallet.

## How to test?

Update your `.env` to include the following:

`VITE_DEQUIPPER_CONTRACT=0x88DD17F0D007e79f1FB2d9332ED4426cA10CCF05`

Go to Settings > Advanced > Dequip. If you need a Testnet Bumpkin minted, message me 👍 

Screenshot 2024-02-26 at 9 03 18 am
Screenshot 2024-02-26 at 8 51 04 am
Screenshot 2024-02-26 at 8 55 18 am

### CODE REVIEW

**Comment by eliasSFL** [comment]
So far the dequipper works fine when testing it on local, but I noticed the wearables goes from those bumpkins to our wallet, instead of going into the farms directly, is it possible for the contract to go directly to the farm address instead?

**Comment by adamhannigan** [comment]
@eliasSFL The dequipper is fairly basic.

The logic to send to the farm would require an extra workflow to be managed (i.e. refreshing the farm after completion). I've opted with sending to the personal wallet for ease of development

File: src/features/island/hud/components/settings-menu/sub-settings/DequipBumpkin.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    setShowSuccess(true);
+  };
+
+  if (showSuccess) {
+    return (
+      <div className="p-2">
+        <Label
+          icon={SUNNYSIDE.icons.confirm}
+          type="success"
+          className="my-2 capitalize"
```
Comment (c-py) [inline src/features/island/hud/components/settings-menu/sub-settings/DequipBumpkin.tsx:83]:
```suggestion
          className="mb-2 capitalize"
```

Looks like `p-2` is already adding the spacing on the top

File: src/features/island/hud/components/settings-menu/sub-settings/DequipBumpkin.tsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    );
+  }
+
+  if (isLoading) {
+    return (
+      <div className="p-2">
+        <p className="loading">{t("loading")}</p>
+      </div>
+    );
+  }
```
Comment (c-py) [inline src/features/island/hud/components/settings-menu/sub-settings/DequipBumpkin.tsx:98]:
```suggestion
  if (isLoading) {
    return (
      <Loading />
    );
  }
```

File: src/features/island/bumpkin/NoBumpkin.tsx [hunk 3]
```diff
@@ -178,8 +187,17 @@ export const NoBumpkin: React.FC = () => {
           type="danger"
           className="mx-auto my-2"
         >{`Level ${requiredLevel} required`}</Label>
+        {missingWearables && (
+          <Label
+            type="danger"
+            className="mx-auto my-2"
+          >{`You cannot deposit an empty Bumpkin`}</Label>
```
Comment (c-py) [inline src/features/island/bumpkin/NoBumpkin.tsx:194]:
```suggestion
          >{t(`You cannot deposit an empty Bumpkin`)}</Label>
```

File: src/features/island/hud/components/settings-menu/sub-settings/DequipBumpkin.tsx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+            );
+          })}
+        </div>
+
+        {missingWearables && (
+          <Label
+            type="danger"
+            icon={lockIcon}
+            className="my-2"
+          >{`Bumpkin has already been dequipped!`}</Label>
```
Comment (c-py) [inline src/features/island/hud/components/settings-menu/sub-settings/DequipBumpkin.tsx:164]:
```suggestion
          >{t(`Bumpkin has already been dequipped!`)}</Label>
```

**Review by c-py — APPROVED** [review]
LGTM, just a few styling conflicts (commented in-line)

**Pull request merged**